### DISCUSSION TITLE
How to make awesome readme profile ?

### DISCUSSION THREAD

**Discussion by d-coder111** [community/community#19958] [Other Feature Feedback, Questions, & Ideas] [answered]
How to make awesome readme profile ?

**Comment by melvincwng** [comment]
@d-coder111, you can use this README.md generator to generate a cool readme profile: 

https://rahuldkjain.github.io/gh-profile-readme-generator/

Hope this helps

**Reply by d-coder111** [reply]
Thanks melvincwng😊🙌

**Comment by manasesjesus** [comment, marked as answer]
Create a repo named as your username (**d-coder111**) and add a README. The info in your readme will be displayed on your profile. You can use [this tool](https://rahuldkjain.github.io/gh-profile-readme-generator/) to generate the contents of your readme, and [this one](https://github.com/anuraghazra/github-readme-stats) to add your statistics. Here's GitHub's step-by-step [profile README tutorial](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme).

Another approach is using GitHub Actions to update your readme whenever an event triggers the action. There are actions such as [this](https://github.com/marketplace/actions/github-update-readme), [this](https://github.com/marketplace/actions/update-action-readme), and [many more](https://github.com/marketplace?type=actions&query=readme+). You can add all the info you want to make it handsome.

You can also code your own action using GitHub GraphQL and REST APIs to get the info you want, and update the readme with that info. It can be displayed in SVG and you can use CSS for colors and animations. Take a look at [my profile](https://github.com/manasesjesus) to see this approach.

Finally, take a look at [this repository](https://github.com/abhisheknaiidu/awesome-github-profile-readme) as it has a collection of readme profiles. You may get inspiration by looking at different profiles.

**Reply by d-coder111** [reply]
Appreciating ur support..Thanks manasesjesus😊

**Reply by i-prince** [reply]
thanks. Really helped.

**Comment by Zi-Tao** [comment]
@d-coder111  These profile templates should be able to help you!
https://github.com/durgeshsamariya/awesome-github-profile-readme-templates
https://github.com/topics/profile-readme
https://github.com/topics/profile-template

And This documentation will guide you in creating a GitHub readme profile!
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme

**Reply by d-coder111** [reply]
Thanks Zi-Tao😊👍
